CONTRIBUTING.md

```diff
@@ -2,11 +2,12 @@
 
 This doc is intended for contributors to `cadence` server (hopefully that's you!)
 
-**Note:** All contributors also need to fill out the [Uber Contributor License Agreement](http://t.uber.com/cla) before we can merge in any of your changes
+>Note: All contributors also need to fill out the [Uber Contributor License Agreement](http://t.uber.com/cla) before we can merge in any of your changes
 
 ## Development Environment
 
-* Go. Install on OS X with `brew install go`.
+* Golang. Install on OS X with `brew install go`. 
+>Note: If running into any compiling issue, make sure you are on a too old version of Golang. 
```

Comment:
Do you mean to use the latest version of Golang?

Reply:
No my intention is to let the developer to upgrade to newer versions if they see compiling issues. The background is when I run the command, because I already installed go so it doesn't do anything, But I still got error because the version on may laptop is 1.11.

Reply:
"make sure you are on a too old version of Golang" -> "make sure you upgrade to the latest stable version of Golang"?

Reply:
I was hesitating not to say "latest". Yeah I am fine with that too, as they have encountered compiling issues, getting the latest makes sense.

Reply:
Done :D